lib/cli/scripts/scan-env.ts

```diff
@@ -262,11 +306,19 @@ function handleError(error) {
     failScript();
 }
 
+/**
+ * Log the error an exit
```

Comment:
probably typo? I guess you meant 'and' not 'an'

Reply:
fixed